hackathon_site/event/jinja2/event/landing.html

```diff
@@ -108,38 +108,39 @@ <h3 class="center" id="countdownTitle"></h3>
                 <div class="m12 center">
                     <h3 class="center">Schedule</h3>
                 </div>
+
                 <div class="col s12 m6 l6">
-                    <h5 class="center">Kickoff</h5>
+                    <h5 class="center">Hardware Pickup 1</h5>
 
                     <div class="scheduleRow">
-                        <p class="scheduleTime">2PM</p>
-                        <p>Saturday, February 6th, 2021</p>
+                        <p class="scheduleTime">6-8PM</p>
+                        <p>Friday, February 18th, 2022</p>
                     </div>
                 </div>
 
                 <div class="col s12 m6 l6">
-                    <h5 class="center">Networking Booths</h5>
+                    <h5 class="center">Hardware Pickup 2</h5>
 
                     <div class="scheduleRow">
-                        <p class="scheduleTime">3-6PM</p>
-                        <p>Saturday, February 6th, 2021</p>
+                        <p class="scheduleTime">10:30AM</p>
```

Comment:
Is there an end time?

Reply:
logistics said "onwards throughout the day" but the row looks all weird if I type too much in the time column